Tasks/GradleV2/task.json

```diff
@@ -1,372 +1,333 @@
 {
-    "id": "8D8EEBD8-2B94-4C97-85AF-839254CC6DA4",
```

Comment:
These changes look redundant

Reply:
Yeah sorry about that, formatting with prettier seems noisy.